localStorage.js

```diff
@@ -0,0 +1,22 @@
+const saveFormData = () => {
+  const formData = {
+    title: document.getElementById('title').value,
+    authur: document.getElementById('authur').value,
+  };
+
+  // Save the entire form data as a single object in local storage
+  localStorage.setItem('formData', JSON.stringify(formData));
+};
+
+// Function to load form data from local storage and pre-fill the form
+const loadFormData = () => {
+  const formData = JSON.parse(localStorage.getItem('formData'));
+  if (formData) {
+    document.getElementById('title').value = formData.title || '';
+    document.getElementById('authur').value = formData.authur || '';
+  }
+};
+  // Add event listeners to save data on input change and load data on page load
+const contactform = document.querySelector('.contact-form');
+contactform.addEventListener('input', saveFormData);
+window.addEventListener('load', loadFormData);
```

Comment:
- [ ] Previous reviewer didn't give this as it is working previously Please make sure that after you add data it should save on the local storage but when I add data it is adding properly but the data is not saved on the local storage as a result after page load the data get loss. Please fix this issue. What I can suggest is that please use the `local storage` logic on main.js so it would be easier for you to do.
### Error
![Screenshot 2023-08-17 144915](https://github.com/tatouroberto/Awesome-Books/assets/81727988/70ad4b2d-fac6-41ec-be08-d754e4cd8737)